models/book.model.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+const { Schema } = require('mongoose');
+
+
+const BookSchema = new Schema({
+  title: {
+    type: String,
+    required: [true, 'Title is required']
+  },
+  author: String,
+  price: Number,
+  img: [],
```

Comment:
why img as array?

Reply:
i might add more images of same product that's why, actually i was thinking of making it an object with keys like default, front and back etc etc